tools/provision/rhel.sh

```diff
@@ -34,18 +54,16 @@ function main_rhel() {
   package python-devel
   package rpm-build
   package ruby
-  sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
   package ruby-devel
   package rubygems
 
   if [[ $DISTRO = "rhel6" ]]; then
     package scl-utils
     package policycoreutils-python
-    package
-    package rhscl-devtoolset-3
     package devtoolset-3
-    sudo scl enable devtoolset-3 bash
+    source /opt/rh/devtoolset-3/enable 
```

Comment:
Sets PATH, LD_LIBRARY_PATH, CC, CXX, INFOPATH, as well as some other python/java variables and shell aliases. There are some changes to our dependency builds in `lib.sh` that make sure the current "selected" compiler chain's variable bubble down.